### ISSUE TITLE
Go to previous/ next error, warning, problem does not read the content to the screen reader

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#174003]
1. turn on a screen reader
2. run `Go to previous/next error, warning, problem`
3. expectation: the error, warning, or problem content will be read to me
4. 🐛 nothing is read

Discovered while trying to get `aria-live` to work for presenting the inline ghost text to a screen reader. It appears that `aria-live` does not work even when it's set to `assertive`. Focusing the tooltip reads it, so maybe that's what we should do here

https://github.com/microsoft/vscode/blob/1028dd25f857c6b735a1870bb63653845786a5d5/src/vs/editor/contrib/gotoError/browser/gotoErrorWidget.ts#L62-L63

**Comment by meganrogge** [comment]
@isidorn would be great if you could pls test and confirm this expectation

**Comment by jrieken** [comment]
/duplicate of https://github.com/microsoft/vscode/issues/148634

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for [similar existing issues](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22accessibility%22). See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
